x-pack/docs/en/security/configuring-stack-security.asciidoc

```diff
@@ -112,3 +125,78 @@ include::{es-ref-dir}/setup/install/security-files-reference.asciidoc[leveloffse
 Additionally, when you use the enrollment token to connect {kib} to a secured {es} cluster, the HTTP layer CA certificate is retrieved from {es} and stored in the
 {kib} `/data` directory. This file establishes trust between {kib} and the {es}
 Certificate Authority (CA) for the HTTP layer.
+
+[discrete]
+[[stack-skip-auto-configuration]]
+=== Cases when security auto configuration is skipped
+When you start {es} for the first time, we try to automatically configure
+security for you. We run some checks to determine whether this is the first time
+that the node is starting. If any of those checks fail, there's a good
+indication that you <<<<manually-configure-security,manually configured security>>,
+or don't want security to be configured automatically. In these cases, we'll
+respect your existing configuration and skip the auto-configuration process. 
+
+[discrete]
+[[stack-existing-environment-detected]]
+==== Existing environment detected
+If certain directories already exist, there's a strong indication that the node
+was started previously. Similarly, if certain files _don't_ exist, or we can't 
+read or write to specific files or directories, then we're likely not running as 
+the user who installed {es} or an administrator imposed restrictions. If any of 
+the following environment checks are true, security isn't configured 
+automatically.
+
+The {es} `/data` directory exists and isn't empty::
+The existence of this directory is a strong indicator that the node was started 
+previously, and we don't want to overwrite your existing configuration.
+
+The `elasticsearch.yml` file doesn't exist (or isn't readable), or the `elasticsearch.keystore` isn't readable::
+If either of these files aren't readable, we can't determine whether {es} security 
+features are already enabled. This state also indicates that we're probably not 
+running as a user who has sufficient privileges.
+
+The {es} configuration directory isn't writable::
+This state likely indicates that an administrator made this directory read-only,
+or that the user who is starting {es} is not the user that installed {es}.
+
+Indices are read-only::
+It's likely that there's not enough disk space, so we can't write the `elastic`
+user password and API key to the `.security` index.
+
+[discrete]
+[[stack-existing-settings-detected]]
+==== Existing settings detected
+If {es} detects any of the following settings, then we'll assume that you want
+to configure security on your own and we won't configure security automatically.
```

Comment:
The following settings are incompatible with the security auto-configuration. If any are present security auto-configuratin is skipped and the node starts normally.

Reply:
I'll change to read:

>The following settings are incompatible with security auto configuration. If any of these settings exist, the node startup process skips configuring security automatically and the node starts normally.